roles/zabbix_agent/defaults/main.yml

```diff
@@ -68,7 +68,7 @@ zabbix_repo_yum:
 
 # Zabbix API stuff
 zabbix_api_server_host: localhost
-zabbix_api_server_port: 80
+# zabbix_api_server_port: 80
```

Comment:
I like having all the vars in defaults/main.yml, is there a reason you want to remove this line?

Reply:
Because if you look at line 28 in the agent role, if `zabbix_api_server_port` is defined, it will always use that one.  It will only default if its not defined (to allow someone to override the port)